packages/commonwealth/client/scripts/views/components/MarkdownEditor/toolbars/CWCreateLinkButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { openLinkEditDialog$, usePublisher } from 'commonwealth-mdxeditor';
+import React, { useCallback } from 'react';
+import CWIconButton from 'views/components/component_kit/new_designs/CWIconButton';
+import { CWTooltip } from 'views/components/component_kit/new_designs/CWTooltip';
+import './CWHeadingButton.scss';
+
+export type CWCreateLinkButtonProps = Readonly<{
+  onClick?: (event: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+}>;
+
+export const CWCreateLinkButton = (props: CWCreateLinkButtonProps) => {
```

Comment:
Basically the general rule is that CW prefix is for the components that are highly reusable and they are part of our component library. Is this is a case with CreateLinkButton? Imo this will be used only in the editor so should be without CW prefix.